### ISSUE TITLE
Rendering SVGs as components by index from array causes strange double path behaviour

### ISSUE THREAD

**Issue by whitep4nth3r** [whitep4nth3r/randomwebsitegenerator.lol#1]
See index.js for the following code:

```javascript
{/* THIS NO WORKY */}
    {svgComponents.filter((svg, index) => svgComponentsIndexArray.includes(index))}

{/* THIS NO WORKY */}
    {svgComponentsIndexArray.map((int) => (
        <div key={int} data-key={int} style={{ height: "20px", width: "20px" }}>
            {svgComponents[int]}
        </div>
    ))}

{/* THIS WORKS */}
    {svgComponents.map((comp, index) => (
        <div key={index} data-key={index} style={{ height: "20px", width: "20px" }}>
            {comp}
        </div>
    ))}
```

Observe that in the top two ways of rendering the svgs, random behaviour happens where there are double path elements rendered inside the svg element, so it appears that some icons are being merged.

Observe that the when we iterate over all components in the svgComponents array, the svgs render correctly.

This is seemingly extremely random.

**Comment by OriginalEXE** [comment]
Should be solved by #2 , I left the explanation there, hope that helps.

**Comment by Ketcap** [comment]
+1 on originalExe and other thing you already have this error on your console `Warning: Text content did not match. Server: "Systms Output Solutionsssss" Client: " Lambdaaa"`
